blog/2021/08/DEQ.md

```diff
@@ -0,0 +1,438 @@
+@def rss_pubdate = Date(2021, 1, 18)
+@def rss = """ Implementing Deep Equilibrium Models in Julia... """
+@def published = "18 January 2021"
+@def title = "Composability in Julia: Implement Deep Equilibrium Models via Neural ODEs"
+@def authors = """Chris Rackauckas, Mike Innes, Yingbo Ma, Jesse Bettencourt, Lyndon White, Vaibhav Dixit"""  
+
+<!-- Translations: [Traditional Chinese](/blog/2019/04/fluxdiffeq-zh_tw) -->
+
+In this blog post we will show you how to easily, efficiently, and
+robustly use steady state solvers with neural networks in Julia.
```

Comment:
it dives very quickly into the topic IMO. Without directly looking into the code at the end, it is not clear what you actually demonstrate in the post. Maybe motivate in the beginning DEQs:

- what are they good for/which tasks can be solved. 
- how can they be described mathematically. (specifically that might be a good point to say it's implicitly defined so you can choose whatever method you want to solve them.. )
- how you would likely solve it with the methods in NLsolve, Bifurcations, etc.
- differences wrt to traditional NN architectures. 

(I guess it's good to make first sure everyone has that information about DEQs)

Then state that you'll implement a DEQ for XYZ problem based on its formulation as a steady state problem of a dynamical system. Highlighting that his needs compositionality between ML and DiffEq tools. Try to make sure that people understand the problem setting well.. 

- make the connection to steady states of ODEs very clear from a mathematical perspective 
- introduce Neural ODEs (at this point it should be rather easy to introduce them. So you can start with a more solid ground and it's clear what a "infinitely small" layer actually is..)
- say what advantages that viewpoint has.

- go to the examples

- for the conclusion: I think it's nice to mention that this basically worked without any new code. (Sure we had to fix some bugs wrt to the type stability of the Steady state solver or the CUDA issue in the adjoints but at least in principle it just requires tools which were already there.) 
In the same spirit, I think writing here many things more like: "this is actually nothing else than a steady state/non-linear solve on top of a neural network" with some more equations would be great.